src/test/java/chess/RookTest.java

```diff
@@ -4,17 +4,20 @@
 import static org.junit.Assert.*;
 
 public class RookTest {
-    @Test
+   @Test
     public void testCreateRook() {
+
```

Comment:
delete this line

Reply:
i made the changes could you review it please?